cloudferry_devlab/cloudferry_devlab/tests/README.md

```diff
@@ -0,0 +1,44 @@
+Directory with tests for Cloudferry's devlab
+==========
+
+## Prerequisite action
+
+Prepare small tenant:
+- vms
+- networks
+- subnets
+- users
+- keypairs
+- tenants
+- modifies tenants quotas
+- upload images and snapshot
+- creates flavors
+- creates volumes
+
+## Usage
+
+Prepare virtual environment to run tests and scripts. You can use the same
+venv as prepared for cloudferry with same requirements and test-requirements
+installed. But also you'll need to install cloudferry_devlab as python package:
+```
+cd cloudferry_devlab
+python setup.py install
+```
+
+Modify config.py if you'll need additional amount of OS objects described above.
+```
+vim cloudferry_devlab/tests/config.py
+```
+
+Prepare the same virtualenv as for Cloudferry and run to create Openstack
+objects on source cloud:
+```
+cd cloudferry_devlab/
+generate_load <configuration.ini>
+```
+
+To execute tests run:
+```
+cd cloudferry_devlab/tests
+nosetests -v
+```
```

Comment:
Please use markdown to highlight filenames (for config.py and test-requirements.txt)